### PR TITLE
Generate flexible server cmdlets

### PR DESCRIPTION
**Pull request by msJinLei**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/MySql/readme.md [hunk 1]
```diff
@@ -47,31 +47,44 @@ In this directory, run AutoRest:
...
 input-file:
   - $(repo)/specification/mysql/resource-manager/Microsoft.DBforMySQL/stable/2017-12-01/mysql.json
   - $(repo)/specification/mysql/resource-manager/Microsoft.DBforMySQL/stable/2017-12-01/ServerSecurityAlertPolicies.json
+  - $(repo)/specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2020-07-01-preview/mysql.json
 module-version: 0.1.0
 title: MySQL
 subject-prefix: 'MySQL'
 
 directive:
-  - from: mysql.json
```
Comment (DaeunYim) [inline src/MySql/readme.md:61]: We need PG flexible server as well..! Could you do the same generation for postgresql as well? 

Single server"input-file:
- Microsoft.DBforPostgreSQL/stable/2017-12-01/postgresql.json
- Microsoft.DBforPostgreSQL/stable/2017-12-01/ServerSecurityAlertPolicies.json
- Microsoft.DBforPostgreSQL/stable/2018-06-01/PrivateEndpointConnections.json
- Microsoft.DBforPostgreSQL/stable/2018-06-01/PrivateLinkResources.json
- Microsoft.DBforPostgreSQL/stable/2020-01-01/DataEncryptionKeys.json

flexible server
input-file:
- Microsoft.DBforPostgreSQL/preview/2020-02-14-preview/postgresql.json

File: src/MySql/readme.md [hunk 2]
```diff
@@ -47,31 +47,44 @@ In this directory, run AutoRest:
...
+    transform: return $.replace(/^CheckVirtualNetworkSubnetUsage_Execute$/g, "VirtualNetworkSubnetUsage_Get")
   - where:
       verb: Set
-      subject: Configuration$|FirewallRule$|VirtualNetworkRule$
+      subject: ^Configuration$|^FirewallRule$|^VirtualNetworkRule$
     set:
       verb: Update
   - where:
-      subject: Database$|SecurityAlertPolicy$|Administrator$|LocationBasedPerformanceTier$|LogFile$|ExecuteCheckNameAvailability$
+      subject: ^Database$|^LocationBasedPerformanceTier$|^LogFile$|SecurityAlertPolicy$|Administrator$
```
[the remark this answers is not in the document]
Comment (msJinLei) [reply src/MySql/readme.md:83]: These cmdlets are not in high priority. Our team has limited resources and so we first implement the most important cmdlets and then gradually add more thing according to the user feedback.

File: src/MySql/readme.md (hunk 1 above)
Comment (msJinLei) [reply src/MySql/readme.md:61]: Yes, we will do the same thing for postgresql. But let's firstly try everything in mysql and ensure them to work and then we are migrate them to postgresql quickly.

File: src/MySql/readme.md [hunk 3]
```diff
@@ -47,31 +47,44 @@ In this directory, run AutoRest:
...
 
 ``` yaml
-branch: ae862b1c090b4c2c951ea46bf97ddbafd6f76d82
+branch: ea6d1725ca9669714cd5f5f969d026b90ecffbd1
 require:
   - $(this-folder)/../readme.azure.noprofile.md
 input-file:
   - $(repo)/specification/mysql/resource-manager/Microsoft.DBforMySQL/stable/2017-12-01/mysql.json
   - $(repo)/specification/mysql/resource-manager/Microsoft.DBforMySQL/stable/2017-12-01/ServerSecurityAlertPolicies.json
+  - $(repo)/specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2020-07-01-preview/mysql.json
```
[the remark this answers is not in the document]
Comment (msJinLei) [reply src/MySql/readme.md:56]: Do you mean to add PrivateEndpointConnections.json, PrivateLinkResources.json, DataEncryptionKeys.json in Microsoft.DBforMySQL?

[the remark this answers is not in the document]
Comment (DaeunYim) [reply src/MySql/readme.md:56]: Oh I meant, including those json files to the input file section to enable the missing single server commands. I can work on it later after adding flexible-server functionality

**Pull request merged**